cmd/mattermost/commands/webhook.go

```diff
@@ -287,10 +291,14 @@ func createOutgoingWebhookCmdF(command *cobra.Command, args []string) error {
 		}
 	}
 
-	if _, err := app.CreateOutgoingWebhook(outgoingWebhook); err != nil {
-		return err
+	createdOutgoing, errOutgoing := app.CreateOutgoingWebhook(outgoingWebhook)
+	if errOutgoing != nil {
+		return errOutgoing
 	}
 
+	CommandPrettyPrintln("Display Name: " + createdOutgoing.DisplayName)
+	CommandPrettyPrintln("Token Id: " + createdOutgoing.Id)
```

Comment:
Why "Token Id"? I think is just "Id" or "Webhook Id".

Reply:
ok, will make the change to Id

Reply:
done @jespino